Airports/EGKK/SMR/Regions.txt

```diff
@@ -845,7 +856,7 @@ N051.09.15.517 W000.10.28.749
 N051.09.12.833 W000.10.27.728
 ;P_Island6
 REGIONNAME Gatwick
-smrGrass N051.09.14.208 W000.10.22.521
+smrGrass 43520 N051.09.14.208 W000.10.22.521
```

Comment:
Remove '43520', this is a colour and refers to smrGrass2 defined in Colours.txt, should just be smrGrass

Reply:
Correction: Remove the line entirely, it is the exact same line as the one above